encoder/vaapiencoder_base.cpp

```diff
@@ -89,12 +89,12 @@ YamiStatus VaapiEncoderBase::start(void)
 void VaapiEncoderBase::flush(void)
 {
     AutoLock l(m_lock);
```

Comment:
Hi @chivakker 
the flush only be call one time. remove it will not improve performance or readbility. Do we really need  remove it?